code/game/machinery/computer/law.dm

```diff
@@ -29,6 +29,10 @@
 		var/input = stripped_input(user, "Please enter the Upload code.", "Uplode Code Check")
 		if(!GLOB.upload_code)
 			GLOB.upload_code = random_code(4)
+		var/use_key_override = FALSE
+		if(input == "666" && upload_module.key_override)
```

Comment:
I don't like this, just give them a popup instead that says 'do you want to override the upload key?'